src/NLog/LayoutRenderers/Wrappers/ObjectPathRendererWrapper.cs

```diff
@@ -0,0 +1,143 @@
+﻿// 
+// Copyright (c) 2004-2019 Jaroslaw Kowalski <jaak@jkowalski.net>, Kim Christensen, Julian Verdurmen
+// 
+// All rights reserved.
+// 
+// Redistribution and use in source and binary forms, with or without 
+// modification, are permitted provided that the following conditions 
+// are met:
+// 
+// * Redistributions of source code must retain the above copyright notice, 
+//   this list of conditions and the following disclaimer. 
+// 
+// * Redistributions in binary form must reproduce the above copyright notice,
+//   this list of conditions and the following disclaimer in the documentation
+//   and/or other materials provided with the distribution. 
+// 
+// * Neither the name of Jaroslaw Kowalski nor the names of its 
+//   contributors may be used to endorse or promote products derived from this
+//   software without specific prior written permission. 
+// 
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE 
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE 
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE 
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR 
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS 
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN 
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) 
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF 
+// THE POSSIBILITY OF SUCH DAMAGE.
+// 
+
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using NLog.Config;
+using NLog.Internal;
+using NLog.Layouts;
+
+namespace NLog.LayoutRenderers.Wrappers
+{
+    /// <summary>
+    /// Render a single property of a object
+    /// </summary>
+    [LayoutRenderer("Object-path")]
+    [AmbientProperty("path")]
+    [AppDomainFixedOutput]
+    [ThreadAgnostic]
+    [ThreadSafe]
```

Comment:
Not sure about the attributes 
    [AppDomainFixedOutput]
    [ThreadAgnostic]
    [ThreadSafe]

Reply:
It can only have `[ThreadSafe]`

Reply:
thanks!

Reply:
@snakefoot the tests says also threadadgnostic?

![image](https://user-images.githubusercontent.com/5808377/63618261-e31fa880-c5eb-11e9-81c2-205f3244e675.png)

Reply:
Yes I guess `[ThreadAgnostic]` is needed if you want `IRawValue`-support.

Reply:
Any suggestions how to solve this?

Reply:
Think I have to retract my statement about "It can only have [ThreadSafe]". Since this is a layout-wrapper then it will be automatically restricted by its Inner-Layout. So it should be okay to have both [ThreadAgnostic] + [ThreadSafe]